src/core/qgsexpression.h

```diff
@@ -129,8 +129,22 @@ class CORE_EXPORT QgsExpression
      * it does not need to be re-parsed.
      */
     QgsExpression& operator=( const QgsExpression& other );
+
+    /**
+     * Create an empty expression
+     */
+    QgsExpression();
```

Comment:
Would be a good idea... I can't even remember why but last week I was looking for a way to do this in current API